### DISCUSSION TITLE
I can't get the STL preview image to work

### DISCUSSION THREAD

**Discussion by SeaGull711** [UnchartedBull/OctoDash#1722] [Q&A] [answered]
Apparently I'm the only one with this problem?  I can't find any advice while searching... how do I get the .stl preview image to show up on OctoDash?  I'm using Cura and I have tried the "print to octoprint" plugin as well as the old school way of uploading.  Any help is appreciated!

**Comment by jneilliii** [comment, marked as answer]
You need a thumbnail plugin in octoprint to get thumbnails in octodash. With cura it's the cura thumbnails plugin.

**Reply by SeaGull711** [reply]
Thank you!!
